tcl/board.tcl

```diff
@@ -245,15 +245,14 @@ proc chooseBoardColors { w {choice -1}} {
     ttk::label $f.bdark -image bp$psize -background [lindex $list 2]
     ttk::label $f.wlite -image wp$psize -background [lindex $list 1]
     ttk::label $f.wdark -image wp$psize -background [lindex $list 2]
-    ttk::button $f.select -text [expr {$count + 1}] -command "updateBoardColors $w $count ; \
+    set fcom "updateBoardColors $w $count ; \
```

Comment:
This works only for the predefined color combinations.
When custom colors are selected the innercoords are not updated
(::board::innercoords should not be added here but instead inside updateBoardColors )